src/widgetastic/browser.py

```diff
@@ -564,12 +568,16 @@ def classes(self, locator, *args, **kwargs):
                         return arr;
                     })(arguments)''')
         else:
-            command = 'return arguments[0].classList;'
-        result = set(self.execute_script(
-            command, self.element(locator, *args, **kwargs),
-            silent=True))
-        self.logger.debug('css classes for %r => %r', locator, result)
-        return result
+            if self.browser_version < '46':
+                command = 'return arguments[0].classList;'
+            else:
+                command = 'return arguments[0].classList.value;'
```

Comment:
Can you please add an explanation why is this happening?

Btw. if this is for firefox, I don't see ifs for firefox :)